catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/converter/TestToIcebergSortOrder.java

```diff
@@ -61,7 +67,9 @@ public void testToSortOrder() {
       if (colName.equals("col_6")
           || colName.equals("col_7")
           || colName.equals("col_8")
-          || colName.equals("col_9")) {
+          || colName.equals("col_9")
```

Comment:
could you add the test about transform? current `isIdentity()` seems not enough

Reply:
> could you add the test about transform? current `isIdentity()` seems not enough

ok

Reply:
updated.